CESC-submission/paper.tex

```diff
@@ -119,46 +119,46 @@ \section{Distributed Resource Markets Design}
+For example, to reduce the risks of the impossibility of fair exchange, micropayments can be employed. That is, instead of paying a large chunk of money for the full service, the payment is divided into small values, each of which is exchanged for a small service amount. For instance, one can pay for retrieving a file in small data chunks instead of paying for the full retrieval all at once. Hence, a service-provider loses a small payment if a client does not pay after receiving a chunk. Similarly, a client loses a small payment if it pays in advance and the service-provider does not send a data chunk in return.
 
 
-On the other hand, to thwart accounting attacks, system designers need to incorporate suitable techniques to prove or confirm resource expenditure, and consequently, confirm that payments are well deserved. In online content delivery, for example, the CAPnet puzzle~\cite{Almashaqbeh19b} can be used to ensure that caches have delivered the requested content. The design of this puzzle follows the second category mentioned above, where solving the puzzle without doing the work is more expensive (resources-wise) than solving it after retrieving the content. In file storage, proof-of-replication~\cite{Fisch19} can be used to prove that a server is still storing the clients’ files with the agreed-upon number of replicas. Here, a detect-and-punish mechanism is used, where failing in providing a correct proof leads to slashing part of the deposit a server pledged when joining the system~\cite{filecoin}. Thus, the type of the provided service directly influences the delivery confirmation mechanism that need to be deployed. \\
+On the other hand, to thwart accounting attacks, system designers need to incorporate suitable techniques to prove or confirm resource expenditure, and consequently, confirm that payments are well deserved. In online content delivery, for example, the CAPnet puzzle~\cite{Almashaqbeh19b} can be used to ensure that caches have delivered the requested content. The design of this puzzle follows the second category mentioned above, where solving the puzzle without doing the work is more expensive (resources-wise) than solving it after retrieving the content. In file storage, proof-of-replication~\cite{Fisch19} can be used to prove that a service-provider is still storing the clients’ files with the agreed-upon number of replicas. Here, a detect-and-punish mechanism is used, where failure to provide a correct proof leads to slashing part of the deposit a service-provider pledged when joining the system~\cite{filecoin}. Thus, the type of the provided service directly influences the mechanisms for delivery confirmation, compensation and punishment that need to be deployed. \\
```

Comment:
where failing in providing a correct proof ->  where failure to provide a correct proof